src/Raven.Server/Documents/Indexes/IndexStore.cs

```diff
@@ -912,11 +970,18 @@ private void OpenIndexesFromRecord(PathSetting path, DatabaseRecord record)
                     OpenIndex(path, definition.Etag, indexPath, exceptions, name);
             }
 
+            if (Directory.Exists(path.FullPath) &&
+                record.AutoIndexes.Count == 0 && record.Indexes.Count == 0)
+            {
+                RecreateIndexes(path, record, exceptions);
```

Comment:
No, this should NOT be called as part of the index store / document store startup.
This should be called as part of the Create Database.